p5-travel-app/src/client/index.js

```diff
@@ -3,14 +3,40 @@
 // styles referenced in html/css
 
 //import functions
+import {getGeoCity} from './js/geoNamesApi';
```

Comment:
not prettiered

Reply:
done in [77541ac](https://github.com/leemoispace/Udacity-Front-End-WebDeveloper/pull/9/commits/77541acca892e61c7f81d34c2e31e59fbb1a7011)